File: app/src/main/java/murait/the/android/mania/PrefManager.java
package murait.the.android.mania;

import android.app.Activity;
import android.content.Context;
import android.content.SharedPreferences;

public class PrefManager {
    public static SharedPreferences pref;
    public SharedPreferences.Editor editor;
    public final static int ADS_SHOW_TIME = 4;

    PrefManager(Activity mContext) {
        if (pref == null) {
            pref = mContext.getSharedPreferences("WebView", Context.MODE_PRIVATE);
        }
    }

    public void addVar() {
        if (pref != null) {
            int var = pref.getInt("XD", 0);
            editor = pref.edit();
            editor.putInt("XD", ++var);
            editor.commit();
        }
    }

    public int getVar() {
        int var = 0;
        if (pref != null) {
            var = pref.getInt("XD", 0);
        }
        return var;
    }
}

File: app/src/main/java/murait/the/android/mania/PrefManager.kt
package murait.the.android.mania

import android.content.Context
import android.content.SharedPreferences

class PrefManager internal constructor(mContext: Context) {
    var editor: SharedPreferences.Editor? = null
    fun addVar() {
        if (pref != null) {
            var tVar = pref!!.getInt("XD", 0)
            editor = pref!!.edit()
            editor?.putInt("XD",++tVar)
            editor?.commit()
        }
    }

    val tVar: Int
        get() {
            var tVar = 0
            if (pref != null) {
                tVar = pref!!.getInt("XD", 0)
            }
            return tVar
        }

    companion object {
        var pref: SharedPreferences? = null
        const val ADS_SHOW_TIME = 10
    }

    init {
        if (pref == null) {
            pref = mContext.getSharedPreferences("WebView", Context.MODE_PRIVATE)
        }
    }
}